queue/queue.go

```diff
@@ -49,32 +49,36 @@ func New(queue Queue, url string, publish, devMode bool, cfg *config.TemporalCon
 func setupConnection(connectionURL string, cfg *config.TemporalConfig) (*amqp.Connection, error) {
 	var (
 		conn *amqp.Connection
+		cert tls.Certificate
+		ca   []byte
 		err  error
 	)
-	if cfg.RabbitMQ.TLSConfig.CACertFile == "" {
+	switch cfg.RabbitMQ.TLSConfig.CACertFile {
+	case "":
 		conn, err = amqp.Dial(connectionURL)
-	} else {
+	default:
 		// see https://godoc.org/github.com/streadway/amqp#DialTLS for more information
 		tlsConfig := new(tls.Config)
 		tlsConfig.RootCAs = x509.NewCertPool()
-		ca, err := ioutil.ReadFile(cfg.RabbitMQ.TLSConfig.CACertFile)
+		ca, err = ioutil.ReadFile(cfg.RabbitMQ.TLSConfig.CACertFile)
 		if err != nil {
-			return nil, err
+			break
 		}
 		if ok := tlsConfig.RootCAs.AppendCertsFromPEM(ca); !ok {
```

Comment:
can be simlified to `if !tlsConfig.RootCAs.AppendCertsFromPEM(ca) {`

Reply:
Good idea